src/main/java/exception/MondayException.java

```diff
@@ -0,0 +1,7 @@
+package exception;
+
+public class MondayException extends Exception {
```

Comment:
What does it mean by "MondayException"? Perhaps you could use a more descriptive name for the exception class.